packages/backend/src/index.ts

```diff
@@ -99,3 +104,88 @@ backend.add(import('@internal/plugin-dynamic-plugins-info-backend'));
 backend.add(import('@internal/plugin-scalprum-backend'));
 
 backend.start();
+
+/**
+ * Adds support for corporate proxy to both 'node-fetch' (using 'global-agent') and native 'fetch' (using 'undici') packages.
+ */
+function configureCorporateProxyAgent() {
```

Comment:
Would not make sense to move this function from index.ts to simplify merging in case of upstream changes?

Reply:
That makes sense - I'll move it.

Reply:
Done in [`96402f9` (#1225)](https://github.com/janus-idp/backstage-showcase/pull/1225/commits/96402f9284094527742190927f4e98404ef0015d)